datasets/gov/lsd/pajamu_ir_gyvenimo_salygos.csv

```diff
@@ -0,0 +1,404 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/lsd/pajamu_ir_gyvenimo_salygos	,,,,,,,,,,,,Lietuvos statistikos departamento namų ūkių bei pavienių asmenų pajamų ir gyvenimo sąlygų statistinio tyrimo duomenys,
+,,,,,,,,,,,,,,
+,,,,Asmens,,,id,,,3,,,Asmenų pajamų ir gyvenimo sąlygų statistinio tyrimo rezultatai,
+,,,,,pb030,string,,,,4,open,,Asmens ID,
+,,,,,hb030,string,NamuUkis,,,4,open,,Namų ūkio ID,

```

Comment:
```suggestion
,,,,,hb030,ref,NamuUkis,,,4,open,,Namų ūkio ID,

```

Reply:
"NamuUkis" nesutampa su modelio pavadinimu, bet siūlymas šitą čia palikti, o modelio pavadinimą keisti (žr. 239 eilutę)

Reply:
Ačiū, pataisyta.